terraform.tf

```diff
@@ -12,7 +12,7 @@ Store infrastructure state in a remote store (instead of local machine):
 https://www.terraform.io/docs/state/purpose.html
 */
 terraform {
-  required_version = "~> 0.11.11"
+  required_version = "~> 0.11.13"
```

Comment:
We're going to have this problem again as soon as a new version of Terraform is released. Would it be better to pin to an exact version, rather than the minor version?

Reply:
Very true, it will happen again.

Seems reasonable to lock it exactly. For gems we'd like an easy and fast way to get bug and security fixes so the minor lock makes sense, Terraform isn't running any code in production per say and it's hard to imagine what an urgent security fix would look like!